var/spack/repos/builtin/packages/seacr/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Seacr(Package):
+    """SEACR (Sparse Enrichment Analysis for CUT&RUN) is intended to call peaks and
+    enriched regions from sparse CUT&RUN or chromatin profiling data in which the
+    background is dominated by zeros"""
+
+    homepage = "https://github.com/FredHutch/SEACR"
+    git = "https://github.com/FredHutch/SEACR.git"
+
+    license("GPL-2.0-only", checked_by="A-N-Other")
+
+    version("1.4-b2", tag="v1.4-beta.2")
+    version("1.3", tag="v1.3")
```

Comment:
The docs say:
> This download method is untrusted, and is not recommended. Although tags are generally more stable than branches, Git allows tags to be moved. Many developers use tags to denote rolling releases, and may move the tag when a bug is patched.

I see that @A-N-Other added:
> Have included the latest official release (@1.3) and the latest beta (@1.4-b2), which has been around since 2021 without being officially released.

Since these tags are pretty old, it is highly unlikely that they'll ever be updated, so the reason given in the documentation does not apply for these tags. I'll try to review.